susemanager-frontend/yarn.lock

```diff
@@ -11594,6 +11594,18 @@ mkdirp@^0.5.3, mkdirp@^0.5.5, mkdirp@~0.5.1:
   dependencies:
     minimist "^1.2.5"
 
+moment-timezone@^0.5.33:
+  version "0.5.33"
+  resolved "https://registry.yarnpkg.com/moment-timezone/-/moment-timezone-0.5.33.tgz#b252fd6bb57f341c9b59a5ab61a8e51a73bbd22c"
+  integrity sha512-PTc2vcT8K9J5/9rDEPe5czSIKgLoGsH8UNpA4qZTVw0Vd/Uz19geE9abbIOQKaAQFcnQ3v5YEXrbSc5BpshH+w==
+  dependencies:
+    moment ">= 2.9.0"
+
+"moment@>= 2.9.0", moment@^2.29.1:
```

Comment:
Since we're adding this here we should remove the other source of moment that we were using before (in case this is not already on your plate).

Reply:
Will be addressed in https://github.com/SUSE/spacewalk/issues/13145 which I can hopefully do once 4.2 is unlocked. 👍